src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+        <label class="form-fild">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-fild">
```

Comment:
Add empty line between siblings. Repeat all over the document
```suggestion
        </label>
        
        <label class="form-fild">
```